exercises/components-in-depth/README.md

```diff
@@ -1,93 +1,92 @@
+commands:
 
-Using `<ng-content>` with `select`, allow each button to receive its own custom label
+```
+ng generate component button-grouping
+ng generate component counter-display
+```
 
 ## Step 3
 
-Add an `@Input` interval property, and a time property both of type `number`
+In `button-grouping.component.ts`, add the following `@Input` property
+- counterValue
 
-- **interval** Acts as the step interval for the timer
-- **time:** Will store the current callculated time
-- **timerId:** Will store the current timer id
+and the following `@Output` EventEmitters
+- onIncrement
+- onDecrement
 
 ## Step 4
 
-Add 4 `@Output` `EventEmitter`s
+Implement the `increment` method to emit the `onIncrement` event containing the 
+`counterValue` incremented by 1.
 
-- **onStart:** Emits when the timer starts
-- **onStop:** Emits when the timer stops
-- **onInterval:** Emits on every timer interval
-- **onReset:** Emits when the timer is reset
+Implement the `decrement` method to emit the `onDecrement` event containing the 
+`counterValue` decremented by 1.
 
 ## Step 5
 
-Implement a `start` method to start a timer using `window.setInterval` and store the returned id in the `timerId` property.
-
-The timer should use the `interval` input value as its interval and should update the `time` property with the latest time.
-The timer should also emit the latest time at every interval using the `onInterval` output.
-
-Current time can be computed as `time += interval / 1000`
+Create two buttons in `button-grouping.component.html` that will call our 
+`increment` and `decrement` methods when pressed.
 
-Optional: The method should also emit `time` if a timer hasn't yet been started using the `onStart` output.
+Use `ng-content` inside the buttons such that we can use projection to pass in
+custom button text.
 
 ## Step 6
 
-Using `window.clearInterval` implement a `stop` method that will stop the current running timer using the stored `timerId` id.
+Place the following snippet into `counter-display.component.ts`
+```ts
+import { Component } from '@angular/core';
 
-Optional: Emit the current time using the `onStop` output.
-
-## Step 7
-
-Implement a `reset` that will reset the `time` to `0` and emit the current time using the `onReset` output
-
-## Step 8
+const defaultCounterValue = 10;
 
-Create a `<app-fare-calculator>` component which will use `<app-timer>` in order to calculate fare based on a given rate.
+@Component({
+  selector: 'app-counter-display',
+  templateUrl: 'counter-display.component.html'
+})
+export class CounterDisplayComponent {
+  counterValue: number;
+  defaultCounterValue: number;
 
-## Step 9
+  ngOnInit() {
+    this.setDefaultValue();
+  }
 
-Add 2 `@Input`s `rate`, and `interval`, and one property `time`
+  setDefaultValue() {
+    this.counterValue = this.defaultCounterValue =
+      Number.isInteger(this.defaultCounterValue) ?
+      this.defaultCounterValue : defaultCounterValue;
+  }
+}
+```
```

Comment:
Same as with my previous comment. Avoid giving them a full solution. We don't want them to copy/paste everything.